server/fleet/errors.go

```diff
@@ -70,22 +101,22 @@ type InvalidArgument struct {
 // one error.
 func NewInvalidArgumentError(name, reason string) *InvalidArgumentError {
 	var invalid InvalidArgumentError
-	invalid = append(invalid, InvalidArgument{
+	invalid.Errors = append(invalid.Errors, InvalidArgument{
```

Comment:
Some duplication can be removed here if do something like `invalid.Append(...)`

Reply:
Good idea. ab707d766.